Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -107,6 +107,7 @@ namespace AzQtComponents
         m_label->setSizePolicy(QSizePolicy::Ignored, QSizePolicy::Preferred);
         m_lineEdit->setSizePolicy(QSizePolicy::Ignored, QSizePolicy::Preferred);
         connect(m_label, &QLabel::linkActivated, this, &BreadCrumbs::onLinkActivated);
+        setMinimumWidth(25);
```

Comment:
I would add a comment explaining the crash and how this fixes it so someone doesn't remove it on accident :)